xdp-dump/tests/test-xdpdump.sh

```diff
@@ -285,6 +290,8 @@ test_exitentry()
 
 test_snap()
 {
+    assert_kernel_version_greater_or_equal 5.7.0
```

Comment:
You should be able to make this test pass with the older kernel, guess only lines 295-296 will probably need to be conditional.